{{cookiecutter.project_shortname}}/docker-compose.full.yml

```diff
@@ -1,4 +1,11 @@
 {% include 'misc/header.py' %}
+#
+# This file is an example of what an Invenio production environment would look like.
```

Comment:
Should we also warn that this doesn't work out of the box? 
Things like this [`./scripts/bootstrap`](https://github.com/inveniosoftware/cookiecutter-invenio-instance/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/Dockerfile#L27) call inside the Dockerfile are going to fail at build time, so people will potentially spend time waiting for the build and then it will fail there.

Reply:
Why is the build script failing?